apps/snapodds/src/app/pages/snap/snap/snap.component.ts

```diff
@@ -208,20 +211,76 @@ export class SnapComponent implements OnInit, OnDestroy {
    * @private
    */
   private loadSportEvents(autoSnap = false): Observable<TvSearchResult> {
+    const snapStartTime = Date.now();
+
     if (!autoSnap) {
       this.appStateStore.dispatch(AppState.SNAP_IN_PROGRESS);
     }
+
     return defer(() => this.webcamComponent.triggerSnapshot()).pipe(
-      switchMap((webcamImage: ManipulatedImage) => {
-        if (autoSnap) {
-          return this.snapOddsFacade.autoSearchSport(webcamImage.blob);
-        } else {
-          return this.snapOddsFacade.searchSport(webcamImage.blob);
-        }
-      })
+      tap(() => this.analyticsService.snapViewSnap()),
```

Comment:
snapViewSnap pushes "Manual Snap" event which is not correct when autoSnap is true.

Reply:
done